### PR TITLE
Document CLI user guide to fix typo for `—custom-syntax`

### PR DESCRIPTION
**Pull request by ybiquitous**
> Which issue, if any, is this issue related to?

None, as it's a documentation fix.

> Is there anything in the PR that needs further explanation?

See: https://github.com/stylelint/stylelint/blob/382961fb6eb67557910a1ff7430a7a180f888bcc/lib/cli.mjs#L81

### CODE REVIEW

File: docs/user-guide/cli.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ stylelint "foo/**/*.css" --config bar/mySpecialConfig.json
 Recursively linting all `.css` files in the `foo` directory using a custom syntax:
 
 ```shell
-stylelint "foo/**/*.css" --customSyntax path/to/my-custom-syntax.js
```
Comment (Mouvedia) [inline docs/user-guide/cli.md:198]: This is probably normalized by meow.
see https://github.com/search?q=stylelint+%22--customSyntax%22&type=code

Comment (ybiquitous) [reply docs/user-guide/cli.md:198]: Yes, both forms work. But, our help prints only `--custom-syntax`.

**Comment by ybiquitous** [comment]
@Mouvedia Can you approve it if you don't find any other problem?

**Review by Mouvedia — APPROVED** [review]

**Comment by ybiquitous** [comment]
Thanks!

**Pull request merged**